jobs/garden/spec

```diff
@@ -84,3 +84,12 @@ properties:
   garden.btrfs_store_size_mb:
     description: "Size in MB of the BTRFS backing store for graph storage (-1 to use all available space)"
     default: -1
+
+  env.http_proxy:
+    description: Http proxy to connect to docker hub 
+
+  env.https_proxy:
+    description: Https proxy to connect to docker hub
```

Comment:
I don't _think_ it should break anything, but I think the `description` strings should be clearer anyway, otherwise it could be misleading in the future.